clients/python-wrapper/villas-python-wrapper.cpp

```diff
@@ -0,0 +1,235 @@
+#include <jansson.h>
+#include <pybind11/pybind11.h>
+#include <uuid/uuid.h>
+#include <villas/node.hpp>
+#include <villas/sample.hpp>
+extern "C" {
+  #include <villas/node.h>
+}
+
+namespace py = pybind11;
+
+class Array {
+  public:
+    Array(unsigned int len) {
+      smps = new vsample *[len]();
+      this->len = len;
+    }
+    ~Array() {
+      for(unsigned int i = 0; i < len; ++i) {
+        sample_decref(smps[i]);
+        smps[i] = nullptr;
+      }
+      delete[] smps;
+    }
+
+    vsample * &operator[](unsigned int idx) {
+      return smps[idx];
+    }
+
+    vsample * &operator[](unsigned int idx) const {
+      return smps[idx];
+    }
+
+    vsample ** get_smps() {
+      return smps;
+    }
+
+    unsigned int size() const {
+      return len;
+    }
+
+  private:
+    vsample **smps;
+    unsigned int len;
+};
+
+//pybind11 can not deal with (void **) as function input parameters,
```

Comment:
Please add spaces between `//` and the following text:

```suggestion
// pybind11 can not deal with (void **) as function input parameters,
```